lib/src/chat/chat.dart

```diff
@@ -674,6 +712,9 @@ class _ChatState extends State<Chat> with ChatComposer, ChatCreateMixin, InviteM
 
   void _closePreview() {
     setState(() {
+      if(widget.sharedData != null){
+        _messageListBloc.dispatch(DeleteCacheFile(path: _filePath));
```

Comment:
Just as a note: We should check if this is fine or if this deletes the file to early (e.g. if we are offline and try to send something or if we have a bigger file). But that's something for the hardening.